src/lib/pages/frontsite/Documents/DocumentDetails/DocumentPanel/DocumentTitle.js

```diff
@@ -35,8 +35,10 @@ export class DocumentTitle extends Component {
           </Grid.Row>
         </Grid>
         <Grid rows={2} textAlign="center">
-          <Grid.Row only="mobile">{metadata.document_type}</Grid.Row>
-          <Grid.Row only="mobile">
+          <Grid.Row className="title-tag" only="mobile">
+            {metadata.document_type}
+          </Grid.Row>
+          <Grid.Row className="title-tag" only="mobile">
```

Comment:
can you put only=mobile on the whole grid? then we would avoid creating custom CSS